app/src/features/observations/BlockObservationPage.tsx

```diff
@@ -172,6 +190,61 @@ const BlockObservationPage = () => {
     return <CircularProgress className="pageProgress" size={40} />;
   }
 
+  const handleSaveAndExit = async (values: ICreateBlockObservationPostRequest) => {
```

Comment:
this function does not use the argument it receives... I dont think values is ever used
can just be:
```
const handleSaveAndExit = async () => {
```